evadb/third_party/databases/mariadb/mariadb_handler.py

```diff
@@ -0,0 +1,148 @@
+        """
+        Initialize the handler.
+        Args:
+            name (str): name of the DB handler instance
+            **kwargs: arbitrary keyword arguments for establishing the connection.
+        """
+        super().__init__(name)
+        self.host = kwargs.get("host")
+        self.port = kwargs.get("port")
+        self.user = kwargs.get("user")
+        self.password = kwargs.get("password")
+        self.database = kwargs.get("database")
+
+    def connect(self):
+        """
+        Establish connection to the database.
+        Returns:
+          DBHandlerStatus
+        """
+        try:
+            self.connection = mariadb.connect(
+                host=self.host,
+                port=self.port,
+                user=self.user,
+                password=self.password,
+                database=self.database,
+            )
+            # Auto commit is off by default.
+            self.connection.autocommit = True
+            return DBHandlerStatus(status=True)
+        except mariadb.Error as e:
+            return DBHandlerStatus(status=False, error=str(e))
+
+    def disconnect(self):
+        """
+        Disconnect from the database.
+        """
+        if self.connection:
+            self.connection.close()
+
+    def check_connection(self) -> DBHandlerStatus:
+        """
+        Method for checking the status of database connection.
+        Returns:
+          DBHandlerStatus
+        """
+        if self.connection:
+            return DBHandlerStatus(status=True)
+        else:
+            return DBHandlerStatus(status=False, error="Not connected to the database.")
+
+    def get_tables(self) -> DBHandlerResponse:
+        """
+        Method to get the list of tables from database.
+        Returns:
+          DBHandlerStatus
+        """
+        if not self.connection:
+            return DBHandlerResponse(data=None, error="Not connected to the database.")
+
+        try:
+            query = f"SELECT table_name as 'table_name' FROM information_schema.tables WHERE table_schema='{self.database}'"
+            tables_df = pd.read_sql_query(query, self.connection)
+            return DBHandlerResponse(data=tables_df)
+        except mariadb.Error as e:
+            return DBHandlerResponse(data=None, error=str(e))
+
+    def get_columns(self, table_name: str) -> DBHandlerResponse:
+        """
+        Method to retrieve the columns of the specified table from the database.
+        Args:
+          table_name (str): name of the table whose columns are to be retrieved.
+        Returns:
+          DBHandlerStatus
+        """
+        if not self.connection:
+            return DBHandlerResponse(data=None, error="Not connected to the database.")
+
+        try:
+            query = f"SELECT column_name as 'name', data_type as 'dtype' FROM information_schema.columns WHERE table_name='{table_name}'"
+            columns_df = pd.read_sql_query(query, self.connection)
```

Comment:
Can you also try running integrations locally? You probably miss a type mapping function https://github.com/georgia-tech-db/evadb/blob/c9ae49890d83bbfdaefcf6cbb54a0e2347dfeb66/evadb/third_party/databases/postgres/postgres_handler.py#L108C13-L108C86.

Reply:
Added the mapping function

Reply:
Thanks. Just double-check, does your integration pass when you run locally?